examples/patchbuilder/component-test.yaml

```diff
@@ -0,0 +1,57 @@
+componentFile: builder.yaml
+rootDirectory: './'
+testCases:
```

Comment:
little bit concerned that this is yet another yaml syntax.

Reply:
understood. if you find places that are confusing, let's make them better!